packages/api/src/external/carequality/command/cq-directory/search-cq-directory.ts

```diff
@@ -112,17 +112,22 @@ export function filterCQOrgsToSearch(orgs: CQOrgBasicDetails[]): CQOrgBasicDetai
   return Array.from(uniqueOrgsById.values());
 }
 
+/**
+ * Returns a list of lower cased URLs to exclude from the Carequality Directory search.
+ */
 function constructGatewayExcludeList(): string[] {
   let excludeList: string[] = [];
   const urlsToExclude = Config.getCQUrlsToExclude();
   if (urlsToExclude) excludeList = urlsToExclude.split(",");
-  return excludeList;
+  return excludeList.map(url => url.toLowerCase());
 }
 
 function hasValidXcpdLink(org: Pick<CQOrgBasicDetails, "urlXCPD">) {
   const urlXCPD = org.urlXCPD;
   if (!urlXCPD) return false;
 
-  const isExcluded = cqExcludeList.some(excludedUrl => urlXCPD.startsWith(excludedUrl));
+  const isExcluded = cqExcludeListLowerCased.some(excludedUrl =>
+    urlXCPD.startsWith(excludedUrl.toLowerCase())
+  );
```

Comment:
should we also lower case the urlXCPD just in case?

Reply:
Good catch! I had `.toLowerCase()` on both, then I lower cased the exclude list and removed the wrong one. 🤦